src/lwc/displayRecords/displayRecords.html

```diff
@@ -0,0 +1,16 @@
+<template>
+    <div>
+        <lightning-card>
+            <lightning-input label="Enter Account Id" onchange={changeHandler}></lightning-input>
```

Comment:
Instead of the onChange of this field, use a separate button(submit button) to perform action onclick ob that button.